e2e/README.md

```diff
@@ -37,3 +37,85 @@ If you have .env file, you can use CI or TEST_PROVIDER environment variables to
 - `TEST_PROVIDER=sauce npm run test:e2e`
 
 Both command do the same, use the saucelabs as a driver, but you get the results and the evidences locally.
+
+### pictures
+
+The results of the tests are written in the terminal, but you can check the created pictures about the elements in the same folder where the tests are located in the ./pics folder, next to the references.
+
+### clean the Tests
+
+If you have too many pics which is confusing you can run a command which clean every pics except of references.
+
+- `npm run clean-test`
+
+
+## Test developement guide
+
+The FEF tests based on visual testing. The basic concept is to take screenshot of every status of elements or pages and make difference to a reference picture.
```

Comment:
I would rather write it like this:
`FeF tests are based on visual testing. The basic concept is to take screenshots of every status of every element or page first and compare the difference between the reference pictures and the actual screenshots.`